vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/model/SequenceActionType.java

```diff
@@ -129,6 +133,52 @@ public Type getArgumentType()
             return argumentType;
         }
     },
+    KEY_DOWN(false)
+    {
+        private final Type argumentType = TypeUtils.parameterize(List.class, String.class);
+
+        @Override
+        public void addAction(Actions actions, Object argument)
+        {
+            perform(argument, (List<String> arg) -> {
+                notEmpty(arg, EMPTY_KEY_LIST_EXCEPTION);
+                for (String kye : arg)
+                {
+                    isTrue(EnumUtils.isValidEnum(Keys.class, kye), WRONG_KEY_LIST_VALUE_EXCEPTION_FORMAT, kye);
+                    actions.keyDown(Keys.valueOf(kye));
+                }
+            });
+        }
+
+        @Override
+        public Type getArgumentType()
+        {
+            return argumentType;
+        }
+    },
+    KEY_UP(false)
+    {
+        private final Type argumentType = TypeUtils.parameterize(List.class, String.class);
+
+        @Override
+        public void addAction(Actions actions, Object argument)
+        {
+            perform(argument, (List<String> arg) -> {
+                notEmpty(arg, EMPTY_KEY_LIST_EXCEPTION);
+                for (String kye : arg)
+                {
+                    isTrue(EnumUtils.isValidEnum(Keys.class, kye), WRONG_KEY_LIST_VALUE_EXCEPTION_FORMAT, kye);
+                    actions.keyUp(Keys.valueOf(kye));
+                }
+            });
+        }
```

Comment:
this code is very similar to `KEY_DOWN` entry, please consider refactoring to extract the same code to a new method and then reuse it

Reply:
refactored